### PR TITLE
Filesystem type extend

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-storage-ng.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Tue Jul 11 15:08:42 UTC 2017 - jreidinger@suse.com
+
+- Added Filesystem::Type#supported_fstab_options and
```
Comment (ancorgs) [inline package/yast2-storage-ng.changes]: It's actually `Filesystems::Type` (plural)

File: src/lib/y2storage/filesystems/type.rb [hunk 2]
```diff
@@ -148,6 +171,24 @@ def to_human_string
...
+      # @return [Array<String>]
+      def supported_fstab_options
+        properties = PROPERTIES[to_sym]
+        default = []
+        return default unless properties
+        properties[:fstab_options] || default
+      end
+
+      # return the best fitting partition id for given filesystem type
+      def default_partition_id
```
Comment (ancorgs) [inline src/lib/y2storage/filesystems/type.rb]: Maybe add a note here like:
```ruby
#
# @note: Take into account that the default partition id can be inappropriate for some
#   partition tables. Consider using {PartitionTable::Base#partition_id_for} to translate
#   the result to a supported id before assigning it to a partition.
```

Other than that, I miss the `@return` tag here

Comment (jreidinger) [inline src/lib/y2storage/filesystems/type.rb]: will add it, thanks for note

**Comment by jreidinger** [comment]
failing due to monkey patch in partitioner, so do not merge without partitioner pr ( WIP)

**Comment by ancorgs** [comment]
Do not create the corresponding partitioner PR. I will handle everything as part of https://trello.com/c/Be6kThDA/647-2-merge-yast2-partitioner-and-yast2-storage-ng

**Comment by jreidinger** [comment]
@ancorgs thanks for doing it. This sprint we have no partitioner PBI to hide it into it. So I have to focus at other parts of code.

**Comment by ancorgs** [comment]
Replaced by #310 (still in progress)

**Pull request closed without merging**